junqo_front/pubspec.yaml

```
@@ -42,7 +42,7 @@ dev_dependencies:
   # activated in the `analysis_options.yaml` file located at the root of your
   # package. See that file for information about deactivating specific lint
   # rules and activating additional ones.
-  flutter_lints: ^4.0.0
+  flutter_lints: ^5.0.0
```

Changer la  version a 4 car la 5 n'est pas  compatible et fait peter les tests
```suggestion
  flutter_lints: ^4.0.0
```